src/peep/staff.h

```diff
@@ -63,6 +63,7 @@ void game_command_set_staff_order(int *eax, int *ebx, int *ecx, int *edx, int *e
 void game_command_set_staff_patrol(int *eax, int *ebx, int *ecx, int *edx, int *esi, int *edi, int *ebp);
 void game_command_fire_staff_member(int *eax, int *ebx, int *ecx, int *edx, int *esi, int *edi, int *ebp);
 
+void staff_get_valid_skins(int* ebx); // Used to give random costume to entertainer
```

Comment:
Comment is not a correct description of this function, nor is it necessary, as the function name is already descriptive enough.